sunpy/map/tests/test_plotting.py

```diff
@@ -127,6 +130,28 @@ def test_rectangle_aia171_top_right(aia171_test_map):
     aia171_test_map.draw_rectangle(bottom_left, top_right=top_right)
 
 
+@pytest.mark.xfail(version.parse(astropy.__version__) >= version.parse("4.2.0"), reason="Astropy >= 4.2 is required")
```

Comment:
```suggestion
@pytest.mark.xfail(version.parse(astropy.__version__) < version.parse("4.2.0"), reason="Astropy >= 4.2 is required")
```

Reply:
Oh :facepalm: I did not see this, my apologies. Was testing something else out.